_test/check-exercises.sh

```diff
@@ -11,9 +11,9 @@ if [ ! -x "./bin/test-exercise" ]; then
    exit 1
 fi
 
-# In DENYWARNINGS mode, do not set -e so that we run all tests.
+# In DENYWARNINGS or CLIPPY mode, do not set -e so that we run all tests.
 # This allows us to see all warnings.
-if [ -z "$DENYWARNINGS" ]; then
+if [ -z "$DENYWARNINGS$CLIPPY" ]; then
```

Comment:
```suggestion
if [ -z "$DENYWARNINGS" ] || [ -z "$CLIPPY" ]; then
```

Equivalent, but I like the more explicit form better.

Reply:
Sure, I will be more explicit. `-z "$DENYWARNINGS$CLIPPY"` is only true if both are empty, so I think it should be && instead of ||.